src/CreeDictionary/CreeDictionary/views.py

```diff
@@ -96,12 +97,14 @@ def index(request):  # pragma: no cover
     """
 
     user_query = request.GET.get("q", None)
+    search_run = None
 
     if user_query:
-        search_results = search_with_affixes(
+        search_run = search_with_affixes(
             user_query,
             include_auto_definitions=should_include_auto_definitions(request),
         )
+        search_results = search_run.serialized_presentation_results()
```

Comment:
Yeah, it's much better to do `.serialized_presentation_results()` here instead of as part of the search run! 👍🏼